### DISCUSSION TITLE
ArgoCD UI Session Timeout

### DISCUSSION THREAD

**Discussion by steddyman** [argoproj/argo-cd#18231] [Q&A]
I have configured the Microsoft OIDC provider im ArgoCD and created PIM groups in AAD for elevated permissions for admin users for a fixed period of time.  This is all working fine, and the roles only grant the admin users when they log in with a user that has requested the role.

However, when the role expires, the ArgoCD UI remains logged in and the user can still perform the admin functions.  I have read that the default timeout for logins to the UI is 24 hours.  Is there a way to change this default timeout so it would be more acceptable for this use case?

**Comment by agaudreault** [comment]
Take a look at https://github.com/argoproj/argo-cd/blob/85ed1b96eaded150318348d2e6f468f11d2f32b3/docs/operator-manual/argocd-cm.yaml#L25
